js/create_project_progress.jsx

```diff
@@ -0,0 +1,83 @@
+var DeployKeyTest = require('./deploy_key_test.jsx');
+
+var CreateProjectProgress = React.createClass({
+
+	checkInterval: false,
+
+	getInitialState: function() {
+		return {
+			complete: false,
+			checkingStatus: false
+		};
+	},
+
+	componentDidMount: function() {
+		this.checkInterval = window.setInterval(this.handleCheckStatus, 10000);
```

Comment:
I meant clearing the timer in the `componentWillUnmount` function. This is just best practice https://facebook.github.io/react/docs/component-specs.html#unmounting-componentwillunmount . My understanding is, it's invalid to work with a component that has been unmounted.